launcher/ui/instanceview/VisualGroup.cpp

```diff
@@ -203,7 +204,8 @@ void VisualGroup::drawHeader(QPainter *painter, const QStyleOptionViewItem &opti
 
         painter->save();
         painter->setFont(font);
-        painter->drawText(textRect, Qt::AlignLeft | Qt::AlignVCenter, text);
+        painter->drawText(textRect, Qt::AlignLeft | Qt::AlignVCenter, text != "" ? text : QObject::tr("Ungrouped"));
```

Comment:
for consistency in the codebase I would propose that you should use `QString::isEmpty()`